third-party/realdds/doc/DDS ICD.md

```diff
@@ -0,0 +1,147 @@
+# realdds Topics
+
+There are 2 topic types in the realdds domain
+* **flexible** - Carries a buffer of data that can be JSON, CBOR or other user custom data
+See https://github.com/IntelRealSense/librealsense/blob/dds/third-party/realdds/include/realdds/topics/flexible/flexible.idl
+* **image** - A buffer of data with some extra fields that describe the image
+See https://github.com/IntelRealSense/librealsense/blob/dds/third-party/realdds/include/realdds/topics/image/image.idl
+
+There are 4 main topics, consisted of these 2 types.
+* **realdds/device-info** of type flexible - The server sends a sample of this topic to inform readers of a connected camera.
+  Some of the details included are the camera model and serial number, this information will be used in the other topic names to ease communication with verious cameras in the network.
+* **realdds/\<model\>/\<serial\>/notification** of type flexible - Notifications are sent from the camera server to the camera user.
```

Comment:
Lets add a link to the structure for every message

Reply:
What do you mean structure?
I mention the message underlying topic type, and in the types section there is a link to the IDL